hybris/bin/custom/airport/airportcore/resources/airportcore-items.xml

```diff
@@ -32,294 +32,505 @@
+                <attributes>
+                    <attribute qualifier="color" type="SwatchColorSet">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                </attributes>
+            </itemtype>
+
+            <itemtype code="DutyCosmeticFragranceVariantProduct" extends="VariantProduct" autocreate="true"
+                      generate="true"
+                      jaloclass="it.uiip.airport.core.jalo.DutyCosmeticColorVariantProduct">
+                <attributes>
+                    <attribute qualifier="fragrance" type="java.lang.String">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                </attributes>
+            </itemtype>
+
+            <itemtype code="DutyFoodsProduct" extends="Product"
+                      generate="true" autocreate="true"
+                      jaloclass="it.uiip.airport.core.jalo.DutyFoodProduct">
+                <attributes>
+                    <attribute qualifier="ingredients" type="java.lang.String">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="calories" type="java.lang.Integer">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="fat" type="java.lang.Integer">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="cholesterol" type="java.lang.Integer">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="sodium" type="java.lang.Integer">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="carbohydrate" type="java.lang.Integer">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="protein" type="java.lang.Integer">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                    <attribute qualifier="geographicalOrigin" type="java.lang.String">
+                        <modifiers optional="false"/>
+                        <persistence type="property"/>
+                    </attribute>
+                </attributes>
+            </itemtype>
+
```

Comment:
Anche in questo caso spostate alcuni attibuti più "specifici" nel DutyFoodsVariantProduct, il base product deve avere attributi più generici